packages/client/src/StreamRegistry.ts

```diff
@@ -94,6 +95,8 @@ export class StreamRegistry implements Context {
         this.chainProvider = this.ethereum.getStreamRegistryChainProvider()
         this.streamRegistryContractReadonly = new Contract(this.config.streamRegistryChainAddress,
             StreamRegistryArtifact, this.chainProvider) as StreamRegistryContract
+        const chainId = this.config.streamRegistryChainRPC?.chainId
+        this.defaultOverrides = this.config.ethereumNetworks?.find((network) => network.chainId === chainId)?.overrides
```

Comment:
Do we need a check here in case `chainId` were undefined for whatever reason.
```suggestion
        if (chainId !== undefined) {
            this.defaultOverrides = this.config.ethereumNetworks?.find((network) => network.chainId === chainId)?.overrides
        }
```

Reply:
I guess the next line would still give out an undefined?